modules/ast/src/main/scala/io/circe/ast/Json.scala

```diff
@@ -97,11 +87,6 @@ sealed abstract class Json extends Product with Serializable {
     }
 
   /**
-   * Attempts to decode this JSON value to another data type.
-   */
-  final def as[A](implicit d: Decoder[A]): Decoder.Result[A] = d(HCursor.fromCursor(cursor))
```

Comment:
I'm a bit sad to see this method go. I know it's redundant but it's also quite convenient. Maybe it could be added back as an extension method using the syntax._ import?